x-pack/plugins/secops/public/containers/errors/translations.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const DATA_FETCH_FAILURE = i18n.translate(
+  'xpack.secops.containers.errors.dataFetchFailureDescription',
```

Comment:
This should probably be `Title` instead of `Description` as per the [i18n guidelines](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types).